### PR TITLE
:neckbeard: Tracking via pixel and access logs

### PR DESCRIPTION
**Pull request by epylinkn**
Database would be a lot easier if you really want to set it up.

I just didn't want to add a server-side component, which I thought might integrate with MTurk better. (Though I've never used it and have no idea.)

### CODE REVIEW

File: sample-access-log.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ac98fade102bebd2aa72486d189728b97bb3238b9e1b2392decabd82cf5a3009 cryptokitty-phrenology.buoydontfloat.com [15/Apr/2019:07:17:08 +0000] 67.85.62.235 - 491B6739EB0DB9BA WEBSITE.GET.OBJECT application.css "GET /application.css HTTP/1.1" 200 - 294 294 12 11 "http://cryptokitty-phrenology.buoydontfloat.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36" - Mx15WWnMCMJJcAOtRwuEVbDbF6Kqa0CttoicqRl9DwYt1XUF0j5KvorEJKKgYeSb6iIOisFxRcI= - - - cryptokitty-phrenology.buoydontfloat.com -
+ac98fade102bebd2aa72486d189728b97bb3238b9e1b2392decabd82cf5a3009 cryptokitty-phrenology.buoydontfloat.com [15/Apr/2019:07:17:08 +0000] 67.85.62.235 - 962BA0AB6AF1B732 WEBSITE.GET.OBJECT i.gif "GET /i.gif?kittySrc=1399245.png&kittyIndex=1&answer=yes HTTP/1.1" 200 - 42 42 7 7 "http://cryptokitty-phrenology.buoydontfloat.com/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.103 Safari/537.36" - m1UKy2riiiNB6Ww9sntDmIHv5dwLZ6SajOc/ZEdDtVvpZ5wemhP6qCglwFPdo1QyNm7MHZa5w+0= - - - cryptokitty-phrenology.buoydontfloat.com -
```
Comment (epylinkn) [inline sample-access-log.txt:9]: Basically, we can parse these from all the logs:

```
"GET /i.gif?kittySrc=1399245.png&kittyIndex=1&answer=yes HTTP/1.1"
```

**Comment by epylinkn** [comment]
@xujenna

File: sample-access-log.txt (hunk 1 above)
Comment (xujenna) [reply sample-access-log.txt:9]: aarrarhgghadghaidogh what is this

Comment (xujenna) [reply sample-access-log.txt:9]: Do you already have data collected? I'm pretty sure we can just use firebase client-side.

**Pull request closed without merging**